includes/REST_API/Stories_Media_Controller.php

```diff
@@ -302,7 +302,7 @@ protected function prepare_items_query( $prepared_args = [], $request = null ) {
 	 * @param WP_REST_Request $request Request object.
 	 * @return WP_REST_Response Response object.
 	 */
-	public function prepare_item_for_response( $post, $request ) {
+	public function prepare_item_for_response( $post, $request ): WP_REST_Response {
```

Comment:
I thought we wanted to avoid these because of the `apply_filters()` usage

Reply:
This method is not returned a filter is it?

Reply:
I can add an ignore on this line then.

Reply:
Fixed